.readme-partials.yaml

```diff
@@ -0,0 +1,109 @@
+introduction: |-
+    > Node.js idiomatic client for [BigQuery Storage](https://cloud.google.com/bigquery).
+
+    The BigQuery Storage product is divided into two major APIs: Write and Read API. 
+    Both BigQuery Storage APIs does not provide functionality related to managing 
+    BigQuery resources such as datasets, jobs, or tables.
+
+    The BigQuery Storage Write API is a unified data-ingestion API for BigQuery. 
+    It combines streaming ingestion and batch loading into a single high-performance API.
+    You can use the Storage Write API to stream records into BigQuery in real time or 
+    to batch process an arbitrarily large number of records and commit them in a single 
+    atomic operation. 
+
+    Read more in our [introduction guide](https://cloud.google.com/bigquery/docs/write-api).
+
+    ```javascript
```

Comment:
Maybe worth explaining what this sample does at a high level in the preface?

Something like "Using a system provided default stream, this code sample demonstrates using the schema of a destination stream/table to construct a writer, and send several batches of row data to the table."